aptos-move/framework/aptos-framework/sources/type_info.move

```diff
@@ -19,11 +19,44 @@ module aptos_framework::type_info {
 
     public native fun type_of<T>(): TypeInfo;
 
+    /// Return `true` if `type_info_1` and `type_info_2` are the same,
+    /// else `false`
+    public fun are_same_type_info(
```

Comment:
nit: How about name this are_equal?

Reply:
Renamed per [`c7813d9`](https://github.com/aptos-labs/aptos-core/pull/2268/commits/c7813d952bf84eb5cdf512e700952de62e024138)